docs/how-to-guides/index.md

```diff
@@ -15,6 +15,7 @@
 - [Applying Clang-Tidy to ROS packages](others/applying-clang-tidy-to-ros-packages.md)
 - [Using Eagleye in Autoware](others/eagleye-integration-guide.md)
 - [Defining temporal performance metrics on components](others/defining-temporal-performance-metrics.md)
+- [An example procedure for adding a new node](others/an-example-procedure-for-adding-and-evaluating-a-new-node.md)
```

Comment:
This title is old. Please update it :pray:

Reply:
I overlooked that. Thank you.
Fixed at https://github.com/autowarefoundation/autoware-documentation/pull/410/commits/823ecb3cedeed8a1bf1ebf3e740b6c6f1ce54153